include/_tia.h

```diff
@@ -0,0 +1,79 @@
+/*****************************************************************************/
+/*                                                                           */
+/* Atari VCS 2600 TIA registers addresses                                    */
+/*                                                                           */
+/* Source: DASM - vcs.h                                                      */
+/*                                                                           */
+/* Florent Flament (contact@florentflament.com), 2017                        */
+/*                                                                           */
+/*****************************************************************************/
+
+union __tia {
+  /* TIA Write Registers */
+  struct {
+    unsigned char VSYNC;
+    unsigned char VBLANK;
+    unsigned char WSYNC;
+    unsigned char RSYNC;
+    unsigned char NUSIZ0;
+    unsigned char NUSIZ1;
+    unsigned char COLUP0;
+    unsigned char COLUP1;
+    unsigned char COLUPF;
+    unsigned char COLUBK;
+    unsigned char CTRLPF;
+    unsigned char REFP0;
+    unsigned char REFP1;
+    unsigned char PF0;
+    unsigned char PF1;
+    unsigned char PF2;
+    unsigned char RESP0;
+    unsigned char RESP1;
+    unsigned char RESM0;
+    unsigned char RESM1;
+    unsigned char RESBL;
+    unsigned char AUDC0;
+    unsigned char AUDC1;
+    unsigned char AUDF0;
+    unsigned char AUDF1;
+    unsigned char AUDV0;
+    unsigned char AUDV1;
+    unsigned char GRP0;
+    unsigned char GRP1;
+    unsigned char ENAM0;
+    unsigned char ENAM1;
+    unsigned char ENABL;
+    unsigned char HMP0;
+    unsigned char HMP1;
+    unsigned char HMM0;
+    unsigned char HMM1;
+    unsigned char HMBL;
+    unsigned char VDELP0;
+    unsigned char VDELP1;
+    unsigned char VDELBL;
+    unsigned char RESMP0;
+    unsigned char RESMP1;
+    unsigned char HMOVE;
+    unsigned char HMCLR;
+    unsigned char CXCLR;
+  };
+  /* TIA Read Registers */
+  struct {
+    unsigned char CXM0P;
+    unsigned char CXM1P;
+    unsigned char CXP0FB;
+    unsigned char CXP1FB;
+    unsigned char CXM0FB;
+    unsigned char CXM1FB;
+    unsigned char CXBLPF;
+    unsigned char CXPPMM;
+    unsigned char INPT0;
+    unsigned char INPT1;
+    unsigned char INPT2;
+    unsigned char INPT3;
+    unsigned char INPT4;
+    unsigned char INPT5;
+  };
+  unsigned char data[45]; // For an unknown reason, this array is required for
```

Comment:
I think that Uz called it "a partially implemented C99 feature".  He wrote only enough cc65 code to allow anonymous bit-fields in structs, and the code in <_vic2.h>.  Therefore, structs don't need any named fields; but, unions need at least one.